src/components/policy/policy_regular/src/policy_table/validation.cc

```diff
@@ -136,6 +136,24 @@ bool ModuleConfig::Validate() const {
       return false;
     }
   }
+
+  if (endpoints.empty()) {
```

Comment:
@dev-gh @LuxoftAKutsan yes, this check is not mandatory because `endpoints.valid()` is called in `ModuleConfig::is_valid()` before `Validate()` call. Removed in https://github.com/smartdevicelink/sdl_core/pull/1623/commits/259302189fc77a42848144353c9acee9207a667d